catalog/cataloger_merge_test.go

```diff
@@ -138,18 +136,19 @@ func TestCataloger_Merge_FromParentConflicts(t *testing.T) {
 	if res.Reference != "" {
 		t.Errorf("Merge reference = %s, expected to be empty", res.Reference)
 	}
-	expectedDifferences := Differences{
-		Difference{Type: DifferenceTypeConflict, Entry: Entry{Path: "/file2"}},
-		Difference{Type: DifferenceTypeConflict, Entry: Entry{Path: "/file5"}},
-	}
-	if res.Summary[DifferenceTypeConflict] != len(expectedDifferences) {
-		t.Fatalf("Merge summary conflicts=%d, expected %d", res.Summary[DifferenceTypeConflict], len(expectedDifferences))
-	}
-	differences, _, err := c.Diff(ctx, repository, "master", "branch1", DiffParams{Limit: -1})
-	testutil.MustDo(t, "diff merge changes", err)
-	if !differences.Equal(expectedDifferences) {
-		t.Errorf("Merge differences = %s, expected %s", spew.Sdump(differences), spew.Sdump(expectedDifferences))
-	}
+	//expectedDifferences := Differences{
```

Comment:
No code in comments please.  E.g. https://kentcdodds.com/blog/please-dont-commit-commented-out-code  Also below.
What does this test test?

Reply:
yep

Reply:
Re-open: no code in comments please.